release/release.go

```diff
@@ -1202,15 +1232,35 @@ func linuxRelease(v version.Version) {
 						} else {
 							log.Printf("restarting curator for %s after failure\n", prefix)
 						}
-						err = runAndStreamStderr(prefix, "./curator", curatorArgs...)
+
+						envOverrides := make(map[string]string)
+						envOverrides["NOTARY_KEY_NAME"] = versionWithSigningInfo.notaryKeyName
+						envOverrides["NOTARY_TOKEN"] = versionWithSigningInfo.notaryToken
+
+						// Remove sensitive information from curator input and log
+						curatorArgsLog := append([]string{}, curatorArgs...)
+						envOverridesLog := make(map[string]string)
+						for k, v := range envOverrides {
+							envOverridesLog[k] = v
+						}
+						apiKeyIdex := findArgIndex(curatorArgsLog, "--api_key")
+						if apiKeyIdex >= 0 {
+							curatorArgsLog[apiKeyIdex] = "[REDACTED]"
+						}
+						envOverridesLog["NOTARY_TOKEN"] = "[REDACTED]"
+						log.Printf("curatorArgs: %v, envOverrides: %v\n", curatorArgsLog, envOverridesLog)
```

Comment:
Small thing, can we add the prefix to the front of this so we can easily associate these args with the curator command output?

Reply:
Good idea! will add.